package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "esplora-api-client",
+  "version": "1.0.0",
+  "description": "Tiny client library for Esplora-based Bitcoin APIs",
+  "keywords": [
+    "api",
+    "bitcoin",
+    "blockstream",
+    "client",
+    "esplora",
+    "mempool",
+    "mempool.space"
+  ],
+  "homepage": "https://github.com/hemilabs/esplora-api-client",
+  "bugs": {
+    "url": "https://github.com/hemilabs/esplora-api-client/issues"
+  },
+  "license": "MIT",
+  "author": "Gabriel Montes <gabriel@bloq.com> (https://github.com/gabmontes)",
+  "files": [
+    "src/**/*.js"
+  ],
+  "main": "src/index.js",
+  "repository": "hemilabs/esplora-api-client",
+  "scripts": {
+    "coverage": "c8 npm test",
+    "format:check": "prettier --check .",
+    "knip": "knip",
+    "lint": "eslint --cache .",
+    "prepare": "husky",
+    "prepublishOnly": "npm run build",
```

Comment:
`build` command is missing in the list. (By the way, is there anything to build? The repo is plain js)

Reply:
Nothing to build 😄